src/sagas/session.js

```diff
@@ -67,7 +67,7 @@ export function* listBuckets(getState, action) {
     // that the credentials are valid :/
     yield put(sessionActions.setAuthenticated());
     // Store this valid server url in the history
-    yield put(historyActions.addHistory(getState().session.serverInfo.url));
+    yield put(historyActions.addHistory(getState().session.auth.server));
```

Comment:
Here we could save `serverInfo.url` in the history? (i.e. not the URL entered but the actual one defined by the server, e.g. after redirect etc.)
What do you think?